packages/components/src/local/organisms/adjudication-messages-list/index.tsx

```diff
@@ -285,44 +285,45 @@ export const AdjudicationMessagesList: React.FC<PropTypes> = ({
   }
 
   useEffect(() => {
-    if (planningMessages.length > 0) {
```

Comment:
@lilitkarapetyan - the only change for this block appears to be the level of indentation?  I think `format document` will remove the indentation.  Or is there another change?